src/PrestaShopBundle/Resources/config/routing/admin/improve/international/state.yml

```diff
@@ -1,6 +1,90 @@
 admin_country_states:
+  path: /country-states
+  methods: [ GET ]
+  defaults:
+    _controller: 'PrestaShopBundle:Admin\Improve\International\State:getStates'
+    _legacy_controller: AdminStates
+
+admin_states_index:
   path: /
   methods: [ GET ]
   defaults:
-    _controller: 'PrestaShopBundle\Controller\Admin\Improve\International\StateController::getStatesAction'
+    _controller: PrestaShopBundle:Admin\Improve\International\State:index
+    _legacy_controller: AdminStates
+#    _legacy_link: AdminStates
```

Comment:
Is this commented line to be kept ? same about other commented lines

Reply:
For `_legacy_link` we keep them commented at first it allows to access the new page (only via its URL) and the legacy controller as well It's useful for QA to compare both pages

The `_legacy_link` is usually enabled when the whole page (list + form) has been completely migrated Often it's done in a last dedicated PR which removes legacy code and enabled the links

Reply:
:+1: